manifests/f/Finebits/MetroCommander/1.3.86.0/Finebits.MetroCommander.yaml

```diff
@@ -10,16 +10,17 @@ Tags:
 - two
 - pane
 - commander
+- folder
+- files
+- folders
+- explorer
 ShortDescription: Metro Commander modern two panes file manager.
 PackageUrl: https://www.microsoft.com/store/apps/9WZDNCRFJB33
 Installers:
 - Architecture: neutral
   InstallerUrl: https://finebitsstorage.z22.web.core.windows.net/metrocommander/packages/MC_1.3.86.0.msix
   InstallerSha256: C30BC2FC1595D4A284C4350A63E7FD74DDDDA44C64D739DB54297B2C4E9D364A
   InstallerType: msix
```

Comment:
```suggestion
  InstallerType: msix
  InstallerSwitches:
    Silent: /S
    SilentWithProgress: /S
```

Reply:
Looks like I overlooked this (my brain thought that `msix` was `msi`), but the switches actually aren't needed here.